saltstack/base/salt/custom/usr/local/bin/custom-ansible.sh

```diff
@@ -8,7 +8,7 @@ yum -y install ansible
 
 # Pull Ansible Playbooks from Git
 mkdir /tmp/ansible-playbooks
-git clone https://github.com/gitowner/ansible-playbooks.git /tmp/ansible-playbooks
+git clone https://github.com/cloudera/ansible-playbooks.git /tmp/ansible-playbooks
```

Comment:
#CB-9301: The existing example link was pointed to gitowner's account in which Cloudera doesn't have any control. Pointed it cloudera.